meshtastic/mesh_interface_runtime/node_presentation.py

```
@@ -0,0 +1,206 @@
+        "hopsAway": "Hops",
+        "channel": "Channel",
+        "lastHeard": "LastHeard",
+        "since": "Since",
+        "isFavorite": "Fav",
+    }
+
+    return name_map.get(name, name)
+
+
+# pylint: disable=too-many-return-statements
+def format_node_field(
```

Ruff (0.15.7)

[warning] Too many return statements (18 > 6)

(PLR0911)